CRM/Core/BAO/FinancialTrxn.php

```diff
@@ -495,4 +495,22 @@ public static function getPartialPaymentWithType($entityId, $entityName = 'parti
     return $value;
   }
 
+  /**
```

Comment:
Our CI doesn't enforce this but the full standard is to have a one line sentence (with a cap letter & a full stop) at the beginning of the comment block. The IDEs like it :-) If you have phpstorm you can turn on phpcs checking within it & it gets real chatty about all these things.

(NB you don't need to re-do the PR for this but if you do have to alter it it would be nice)